src/core/ColorPicker/ColorPicker.test.tsx

```diff
@@ -920,8 +920,85 @@ it('should render color picker when alpha is false', () => {
   const sliderDot = sliderThumbs[0] as HTMLElement;
   expect(sliderDot).toBeTruthy();
 
+  // Should handle hue slider change
   fireEvent.keyDown(sliderDot, { key: 'ArrowRight' });
-  expect(onSelectionChanged).toHaveBeenCalledWith(
+  expect(handleOnChange).toHaveBeenCalledWith(
     ColorValue.create({ h: 1, s: 100, l: 50 }),
   );
 });
+
+it('should handle hex input when alpha is false', () => {
+  const handleOnChange = jest.fn();
+  const selectedColor = ColorValue.create({ h: 0, s: 100, l: 50 });
+  const { container } = render(
+    <ColorPicker
+      showAlpha={false}
+      onChangeComplete={handleOnChange}
+      selectedColor={selectedColor}
+    >
+      <ColorInputPanel defaultColorFormat={'hex'} />
+    </ColorPicker>,
+  );
+
+  const inputs = container.querySelectorAll('input');
+  expect(inputs.length).toBe(1);
+  const input = inputs[0] as HTMLInputElement;
+  expect(input).toBeTruthy();
+
+  fireEvent.change(input, { target: { value: '#FF6200' } });
+  fireEvent.keyDown(input, { key: 'Enter' });
+
+  expect(handleOnChange).toHaveBeenCalledWith(ColorValue.create('#FF6200'));
+});
+
+it('should handle hsl input when alpha is false', () => {
+  const handleOnChange = jest.fn();
+  const selectedColor = ColorValue.create({ h: 0, s: 100, l: 50 });
+  const { container } = render(
+    <ColorPicker
+      showAlpha={false}
+      onChangeComplete={handleOnChange}
+      selectedColor={selectedColor}
+    >
+      <ColorInputPanel defaultColorFormat={'hsl'} />
+    </ColorPicker>,
+  );
+
+  const inputs = container.querySelectorAll('input');
+  expect(inputs.length).toBe(3);
+  const input = inputs[0] as HTMLInputElement;
+  expect(input).toBeTruthy();
+
+  fireEvent.change(input, { target: { value: '100' } });
+  fireEvent.keyDown(input, { key: 'Enter' });
+
+  expect(handleOnChange).toHaveBeenCalledWith(
+    ColorValue.create({ h: 100, s: 100, l: 50 }),
+  );
+});
+
+it('should handle rgb input when alpha is false', () => {
```

Comment:
This file is getting huge. Maybe move all (new and old) cases that are only testing the inputs into a new file `ColorInputPanel.test.tsx`